source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -262,6 +262,20 @@ Run the following test(s) on MongoDB 4.4+.
              mode: "off",
          });
 
+Heartbeat Tests
+~~~~~~~~~~~~~~~
+
+1. Test that a failure to create a monitoring connection results in the emission of a
+   ``ServerHeartbeatStartedEvent`` and a ``ServerHeartbeatFailedEvent``
+
+   #. Create a client with ``serverSelectionTimeoutMS: 500`` and listen to ``ServerHeartbeatStartedEvent`` and ``ServerHeartbeatFailedEvent``, pushing the event names to an array.
+
+   #. Attempt to connect client to a local port that is not listening for connections. e.g. ``mongodb://localhost:9999``
+
+   #. Assert that the event array has the following contents in the same order:: 
```

Comment:
Since SDAM can try to connect multiple times, I would say: 
"Assert that the first two events in the array are::"